src/CamOverlayDrawingAPI.ts

```diff
@@ -198,7 +198,6 @@ export class CamOverlayDrawingAPI extends EventEmitter {
             });
             this.ws.on('error', (error: Error) => {
                 this.reportError(error);
-                reject(error);
```

Comment:
kdyz tohle smazes a nekdo bude cekat na connect..zustane v pripade chyby cekat na vzdy..bud to musime reportovat a musi si to handlovad ten co na to awaituje nebo connect nesmi vracet promis aby na to nekdo mohl cekat jinak nez pred emit(open) udalost